### DISCUSSION TITLE
Are dynamic theme YouTube tweaks needed anymore?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [darkreader/darkreader#6829] [Ideas]
The DR dynamic theme has many rules/styles for YouTube.

YouTube now has a native dark theme that will auto-detect if the browser is set to use a dark theme.

Are all those DR dynamic theme tweaks still needed for YouTube, or can some/all be removed?

**Comment by Gusted** [comment]
> Are all those DR dynamic theme tweaks still needed for YouTube, or can some/all be removed?

Most of them were added due to a stupid change from Youtube's team, it's mostly for chromium users but most of them could be obsolete but it's a pain in the ass to go to each type of webpage and check if the fix is still needed. So we mostly don't touch thing, if it just works and not break anything.
